npm/builder.js

```diff
@@ -67,3 +69,34 @@ Builder.prototype.removeDistDirSync = function () {
   bfs.removeSync('dist');
   bfs.mkdirsSync('dist/templates');
 };
+
+Builder.prototype.examples = function (callback) {
+  const builder = this;
+
+  async.series([
+    callback => builder.build(callback), // Build
+    callback => builder.compileExamples(callback), // Compile examples
+  ], () => {
+    log.info(`
+In order to visualize the result, a local HTTP server must be started within the root of this project otherwise you will have cross-origin issues.
```

Comment:
I don't think it applies in this context

Reply:
Removed